### PR TITLE
#6006 issue: Update p5.Color.js

### LINKED ISSUE #6006: Document parameters of p5.Color constructor
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [X] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### p5.js version

1.5.0 (cloudflare CDN)

### Web browser and version

all of them

### Operating System

n/a

### Steps:

```html
<!DOCTYPE html>

<html lang="en">
<head>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.5.0/p5.js"></script>
</head>
<body>
  <script>
    p5.Color(0)
  </script>
</body>
</html>
```

Yields the following error:

```
Uncaught TypeError: this._storeModeAndMaxes is not a function
    Color https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.5.0/p5.js:53391
    <anonymous> http://localhost:8000/index.html:9
[p5.js:53391:16](https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.5.0/p5.js)
    Color https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.5.0/p5.js:53391
    <anonymous> http://localhost:8000/index.html:9
```

### PR DESCRIPTION
**Pull request by sawaisinghh**
Resolves #6006 

 Changes:
Document parameters of p5.Color constructor

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

File: src/color/p5.Color.js [hunk 1]
```diff
@@ -24,9 +24,17 @@ import color_conversion from './color_conversion';
  * We also cache normalized, floating-point components of the color in various
  * representations as they are calculated. This is done to prevent repeating a
  * conversion that has already been performed.
+ * 
+ * <a href="#/p5/color">color()</a> is the recommended way to create an instance
+ * of this class. But, if we use constructor then, it is required to use parameters.
```
[the remark this answers is not in the document]
Comment (sawaisinghh) [reply src/color/p5.Color.js:29]: Yeah, good suggestion

File: src/color/p5.Color.js [hunk 2]
```diff
@@ -24,9 +24,17 @@ import color_conversion from './color_conversion';
...
  * conversion that has already been performed.
+ * 
+ * <a href="#/p5/color">color()</a> is the recommended way to create an instance
+ * of this class. But, if we use constructor then, it is required to use parameters. 
  *
  * @class p5.Color
  * @constructor
+ * @param {p5} [pInst]                      pointer to p5 instance.
+ *                                 
+ * @param {p5.color|Number[]|String} vals   an array or object containing the color
```
[the remark this answers is not in the document]
Comment (sawaisinghh) [reply src/color/p5.Color.js:35]: Okay, I will correct it.

**Review by davepagurek — APPROVED** [review]
Thanks for making the changes, looks great!

**Comment by davepagurek** [comment]
Looks like the linting test failed because there are trailing spaces, I'll just remove those

**Comment by davepagurek** [comment]
@all-contributors please add @sawaisinghh for docs

**Comment by davepagurek** [comment]
https://github.com/all-contributors please add @sawaisinghh for documentation

**Comment by davepagurek** [comment]
@all-contributors please add @sawaisinghh for documentation

**Comment by davepagurek** [comment]
@all-contributors please add @sawaisinghh for documentation

**Comment by davepagurek** [comment]
(sorry for the noise! had to fix some json issues before it'd work.)

**Pull request merged**